### PR TITLE
Remove updates on tensors support specifying only a subset of mapped …

### PR DESCRIPTION
**Pull request by geirst**
…dimensions.

Also improve doc on other tensor update operations and link to system tests for more examples.

@arnej27959 please review

### CODE REVIEW

File: documentation/reference/document-json-format.html [hunk 1]
```diff
@@ -941,6 +944,25 @@ <h3 id="tensor-add">Add tensor cells</h3>
 In this example, cell <code>{"x":"b","y":"2"}</code> will implicitly be set will value 0.0.
 </p>
 
+<p>
+Prefer the <i>block short form</i> for mixed tensors instead. This also avoids the problem where cells with indexed dimensions are not specified:
```
Comment (kkraune) [inline documentation/reference/document-json-format.html:948]: &lt;i&gt; is deprecated in html5 afaik, &lt;em&gt; is a good alternative

**Comment by kkraune** [comment]
It is a great idea to link to system tests, creates less doc :-)

File: documentation/reference/document-json-format.html (hunk 1 above)
Comment (geirst) [reply documentation/reference/document-json-format.html]: Fixed

**Review by arnej27959 — APPROVED** [review]
👍

**Pull request merged**